mantle/kola/tests/misc/network.go

```diff
@@ -56,28 +56,6 @@ func init() {
 		Distros:     []string{"rhcos"},
 		Platforms:   []string{"qemu-unpriv"},
 	})
-	// This test follows the same network configuration used on https://github.com/RHsyseng/rhcos-slb
-	// with a slight change, where the MCO script is run from ignition: https://github.com/RHsyseng/rhcos-slb/blob/main/setup-ovs.sh.
-	// and we're using veth pairs instead of real nic when setting the bond
-	register.RegisterTest(&register.Test{
-		Run:         NetworkBondWithDhcp,
-		ClusterSize: 0,
-		Name:        "rhcos.network.bond-with-dhcp",
-		Timeout:     20 * time.Minute,
-		Distros:     []string{"rhcos"},
-		Platforms:   []string{"qemu-unpriv"},
-	})
-	// This test follows the same network configuration used on https://github.com/RHsyseng/rhcos-slb
-	// with a slight change, where the MCO script is run from ignition: https://github.com/RHsyseng/rhcos-slb/blob/main/setup-ovs.sh.
-	// and we're using veth pairs instead of real nic when setting the bond
-	register.RegisterTest(&register.Test{
-		Run:         NetworkBondWithRestart,
-		ClusterSize: 0,
-		Name:        "rhcos.network.bond-with-restart",
-		Timeout:     20 * time.Minute,
-		Distros:     []string{"rhcos"},
-		Platforms:   []string{"qemu-unpriv"},
-	})
```

Comment:
Hmm, not sure I follow why we're removing these tests. Are they no longer testing relevant scenarios? We shouldn't remove them just because they're failing.

Reply:
We are no longer configuring the bond using these scripts - they are now configured using kubernetes-nmstate manifests.
The new test added represent the scripts that are currently used, and have been approved by our client.

Reply:
OK, can you add that context to the commit message?

Reply:
Sure😃

Reply:
DONE